imports/ui/components/navigation.js

```diff
@@ -3,17 +3,30 @@ Template.header.events({
   'click .menubar-toggle': function (e) {
     e.preventDefault();
     if (!$('.device-xs').is(':visible')) {
-      $('body').toggleClass('menubar-pin');
+      handleToggle();
     } else {
       handleToggleMob();
     }
+  },
+  "click #logout": function(e) {
+    e.preventDefault();
+    $('body').removeClass('menubar-pin');
   }
 });
 
 Template.header.onRendered(function(){
   treatMenuHover();
 });
 
+function handleToggle() {
```

Comment:
No, because menubar on hover uses "menubar-visible" as class.
"menubar-pin" is used just when the toggle button is clicked.